### PR TITLE
R4R: added confirm sig client storage

### PR DESCRIPTION
**Pull request by colin-axner**
Client side storage of confirmation signatures. 

Opening as pr to my own branch since it is easier to review that way. I changed keystore directory to store and add a sigs.go file for storing confirm sigs. I am not sure if it would be better to move client/store dir into our store/ dir, since one is for application storage and the other is for client storage. 

.plasmacli will now have: data/, keys/, plasma.toml
data/ has accounts.ldb and signatures.ldb

You might need to rm your current .plasmacli directory before installing this branch

Upgrade to sdk 0.32.0

### CODE REVIEW

File: server/plasmad/main.go [hunk 1]
```diff
@@ -75,9 +73,3 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer) abci.Application
 		app.SetPlasmaOptionsFromConfig(plasmaConfig),
 	)
 }
-
-// non-functional
-func exportAppState(logger log.Logger, db dbm.DB, traceStore io.Writer, _ int64, _ bool) (json.RawMessage, []tmtypes.GenesisValidator, error) {
```
Comment (AdityaSripal) [inline server/plasmad/main.go]: Should be implemented before Public testnet IMO

**Comment by AdityaSripal** [comment]
Looks like there's no way to send sigs through the sidechain yet? Im assuming this is just implementing client side storage part?

File: client/plasmacli/keys/update.go [hunk 2]
```diff
@@ -31,10 +32,12 @@ Usage:
 		name := args[0]
 
 		updatedName := viper.GetString(nameF)
```
Comment (hamdiallam) [inline client/plasmacli/keys/update.go]: should we not check that the updatedName is not ""

Comment (hamdiallam) [reply client/plasmacli/keys/update.go]: ah, nvm. If the flag is omitted, then its a pw update

**Pull request merged**